libs/velo-external-db-core/src/index.ts

```diff
@@ -65,6 +77,7 @@ export class ExternalDbRouter {
 
     isInitialized(connector: DbConnector) {
         if (!connector.initialized) {
+            this.logger.error('Connector must be initialized before being used')
             throw new Error('Connector must be initialized before being used')
```

Comment:
maybe we should catch the error in some middleware (as I think we already do) and log them there, I don't think we want to log before every throw